docs/languages/main-trio.md

```diff
@@ -1,5 +1,5 @@
-# 🔧 BCL API reference documentation
+# 🔧 The main trio: C#, F# and VB.NET
```

Comment:
Per LCA guidelines, it actually needs to be Visual Basic. We don't own the trademark for VB.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
From: Phillip Carter <notifications@github.com>
Sent: Monday, November 30, 2015 1:15:21 PM
To: dotnet/core-docs
Cc: Steve Hoag
Subject: Re: [core-docs] Update TOC and directory structure (#175)


In docs/languages/main-trio.md<https://github.com/dotnet/core-docs/pull/175#discussion_r46204694>:

> @@ -1,5 +1,5 @@
> -# ? BCL API reference documentation
> +# ? The main trio: C#, F# and VB.NET


Should we call it VB.NET or VB? @richlander<https://github.com/richlander> suggested that I change VB.NET to VB in the async docs a while back.

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/core-docs/pull/175/files#r46204694>.